Formula/wine.rb

```diff
@@ -239,22 +246,28 @@ def install
         system "make", "install"
 
         %w[libcrypto libssl].each do |libname|
-          system "lipo", "-create", "#{dirs.first}/#{libname}.1.0.0.dylib",
-                                    "#{dirs.last}/#{libname}.1.0.0.dylib",
-                         "-output", "#{libexec}/lib/#{libname}.1.0.0.dylib"
+          MachO::Tools.merge_machos("#{libexec}/lib/#{libname}.1.0.0.dylib",
+                                    %W[
+                                      #{dirs.first}/#{libname}.1.0.0.dylib
+                                        dirs.last}/#{libname}.1.0.0.dylib
```

Comment:
This doesn't look quite right. Is `dirs.last}` here supposed to be `#{dirs.last}`?